manual/de/Help_ch_Tools_Assistants.xml

```diff
@@ -4188,136 +4188,196 @@
+                  dass eine terminierte Buchung im <xref linkend="sched-editor" /> angelegt ist, allerdings momentan
+                  keine Vorgänge zur Ausführung anstehen.
                 </para>
               </listitem>
             </varlistentry>
           </variablelist>
         </listitem>
       </varlistentry>
+<!-- FIXME source: new item -->
+      <varlistentry>
+        <term><guilabel>Wert</guilabel></term>
+
+        <listitem>
+          <para>Ist für die automatische Erstellung noch die Angabe des Buchungswerts notwendig, so wird das mit
+            dem Hinweis <emphasis>(Wert benötigt)</emphasis> angezeigt. Dies kann z.B. bei einer
+            Buchung zwischen zwei Konten mit unterschiedlichen Währungen notwendig sein.
+          </para>
+
+          <para>Sie können mit einem Doppelklick auf den Hinweistext ein Eingabefeld öffnen und einen Wert für
+            die Buchung eintragen. Das passiert ebenfalls, wenn Sie mit <guilabel>OK</guilabel> die
+            terminierte Buchung in ein Kontobuch übernehmen wollen.
+          </para>
+        </listitem>
+      </varlistentry>
     </variablelist>
 
-    <para>At the bottom of the window there are two buttons:
-    </para>
+    <variablelist>
+      <title>Am unteren Rand des Fensters, in der Aktionsleiste, befinden sich weitere Steuerelemente:</title>
 
-    <itemizedlist>
-      <listitem>
-        <para>The <guibutton>Cancel</guibutton> button is used to exit the <guilabel>Since Last Run</guilabel>
-          window without creating the transactions scheduled to be entered.
-        </para>
-      </listitem>
+      <varlistentry>
+        <term>Erzeugte Buchungen durchsehen</term>
 
-      <listitem>
-        <para>The <guibutton>Ok</guibutton> button closes the <guilabel>Since Last Run</guilabel> window and
-          applies the pending operations.
-        </para>
-      </listitem>
-    </itemizedlist>
+        <listitem>
+          <para>Wenn diese Option aktiviert ist, während eine oder mehrere Buchungen <guilabel>Zu
+            erstellen</guilabel> sind, wird durch Drücken von <guibutton>OK</guibutton> ein neues
+            Registerfenster geöffnet, in dem die Details der automatisch von &app; erstellten
+            Buchungsteile angezeigt werden und von Ihnen bearbeitet werden können.
+          </para>
+        </listitem>
+      </varlistentry>
 
-    <para>At the bottom right of the <guilabel>Since Last Run</guilabel> window there is also an option that
-      allows to <guilabel>Review created transactions</guilabel>. If this option is enabled when
-      there are one or more transactions <guilabel>To-create</guilabel>, pressing
-      <guibutton>OK</guibutton> will open a register window in which are shown the details of the
-      scheduled transactions splits entered automatically by &app;.
-    </para>
+      <varlistentry>
+        <term>Hilfe</term>
+
+        <listitem>
+          <para>Öffnet die Hilfe für diesen Dialog.
```

Comment:
Öffnet diese Seite im Hilfe-Fenster.
Ermöglicht diese Seite [der Hilfe] aus dem Dialog zu öffnen.
… oder in der Art …

Möglicherweise eine Entität wert.